.mockery.yaml

```diff
@@ -58,6 +58,7 @@ packages:
   github.com/Dynatrace/dynatrace-operator/pkg/webhook:
```

Comment:
In the case of the v2 webhook the initContainer is still called `dynatrace-operator` is this correct?

Reply:
```
Init Containers:
  dynatrace-operator:
    Container ID:  containerd://67b1210dc324edae1484e268d8a7352ffc873dc26e4c71643bad769f4e332e27
    Image:         quay.io/dynatrace/dynatrace-bootstrapper:snapshot
    Image ID:      quay.io/dynatrace/dynatrace-bootstrapper@sha256:753a3cdeea99b3310380edb5513892b4613ab66e9b6869c2f0bedf22e528323d
    Port:          <none>
    Host Port:     <none>
    Args:
      --suppress-errors
      --source=/opt/dynatrace/oneagent
      --target=/mnt/bin
      --install-path=/opt/dynatrace/oneagent-paas
      --technology=php
```

args look good, they are set correctly it seems

Reply:
It should always be named `dynatrace-operator` 😅